wekesa931/Questioner
# [finishes #163100401] creates the vote api endpoint

#### What does this PR do?
It creates the upvote and downvote api endpoint
#### Description of Task to be completed?
To create the upvote and downvote api endpointpath:
```
PATH         /api/v1/question<question>id>/upvote
PATH         /api/v1/question<question>id>/downvote
```
This endpoint captures the required details of the user which are:
#### How should this be manually tested?
clone the repository to your local machine
```
$ git clone https://github.com/wekesa931/Questioner.git
```
Set up the virtual environment
```
$ pip install virtualenv
$ virtualenv venv
$ venv source bin/activate
```
run the application
```
$ python run.py
```
Test the endpoint in postman as follows:
1. In postman, go to path
```
 POST      /api/v1/user/auth/signup
```
2. Add the details of the user
3. Check the user has been added with the GET method
```
PATH         /api/v1/question<question>id>/upvote
PATH         /api/v1/question<question>id>/downvote
```

#### What are the relevant pivotal tracker stories?
    #163100401
